docs/before/analysis.md

```diff
@@ -0,0 +1,155 @@
+!!! note "Quicklink"
+    - **btx** - [btx Github repository][1]
+    - **documentation** - [btx code documentation][2]
+
+# Routine beamtime analysis
+
+LCLS offers several tools to help you analyze your experimental data. 
+The chart below attempts at articulating them based on whether they are more generic tools like `psana` or more experiment specific like `OM` 
+and whether they are mostly used for live analysis during beamtimes like `AMI` or for slightly delayed feedback or even offline like `smalldata` and `btx`. 
+The `eLog` provides a unifying entrypoint from your browser.
+
+| ![Data Analysis Tools](images/data_analysis_tools.png) | 
+|:--:| 
```

Comment:
Maybe we can consider different figure designs in the future. I think I get what you are going for but find it a little confusing now. E.g. it suggests that smalldata and AMI are the same tool but on opposite sides of the use cases?

Maybe we could have  a tree of some sort that builds from psana and one branch goes to offline and one to online?

Not sure exactly. Let's leave it as is for now.